docker/mongodb/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mongo:3.6
```

Comment:
Let's use the most recent version of mongo

Reply:
Raised separate PR for this: https://github.com/kanisterio/kanister/pull/274

Reply:
[minor] We don't need to copy the license in this directory. we can just include the copyright header.

Reply:
I am not sure if this PR is valid anymore because of #274